app/src/main/java/com/humara/nagar/ui/report/ReportFragment.kt

```diff
@@ -23,23 +59,281 @@ class ReportFragment : BaseFragment() {
         container: ViewGroup?,
         savedInstanceState: Bundle?
     ): View {
-        val notificationsViewModel =
-            ViewModelProvider(this).get(ReportViewModel::class.java)
 
         _binding = FragmentReportBinding.inflate(inflater, container, false)
-        val root: View = binding.root
 
-        val textView: TextView = binding.textDashboard
-        notificationsViewModel.text.observe(viewLifecycleOwner) {
-            textView.text = it
+        initViewModelObservers()
+        initView()
+
+        binding.inputLocation.setLayoutListener {
+            checkForLocationPermission()
         }
-        return root
+
+        binding.addBtn.setOnClickListener {
```

Comment:
Add the click listener to whole `addImageLayout` instead of button only

Reply:
Fixed.